src/plugins/gui/scenes/AddressFormScene.tsx

```diff
@@ -77,16 +187,56 @@ export const AddressFormScene = React.memo((props: Props) => {
       <KeyboardAwareScrollView keyboardShouldPersistTaps="handled" extraScrollHeight={theme.rem(2.75)} enableAutomaticScroll enableOnAndroid>
         <SceneHeader title={headerTitle} underline withTopMargin />
         <EdgeText style={styles.formSectionTitle}>{s.strings.home_address_title}</EdgeText>
-        <GuiFormField fieldType="address" autofocus label={s.strings.address_line_1} value={formData.address} onChangeText={handleChangeAddress} />
-        <GuiFormField fieldType="address2" label={s.strings.address_line_2} value={formData.address2} onChangeText={handleChangeAddress2} />
-        <GuiFormField fieldType="city" label={s.strings.address_city} value={formData.city} onChangeText={handleChangeCity} />
-        <GuiFormField fieldType="state" label={s.strings.address_state_province_region} value={formData.state} onChangeText={handleChangeState} />
+        <GuiFormField
+          fieldType="address"
+          autofocus
+          label={s.strings.address_line_1}
+          value={formData.address}
+          fieldRef={rAddressInput}
+          onChangeText={handleChangeAddress}
+          onFocus={handleShowAddressHints}
+          onBlur={handleHideAddressHints}
+        />
+        {isHintsDropped ? (
+          <View style={Platform.OS === 'ios' ? styles.dropContainer : styles.dropContainerAndroid}>
+            <ScrollView keyboardShouldPersistTaps="always" nestedScrollEnabled>
```

Comment:
Using a FlatList or FlashList. You're re-inventing a FlatList using a scrollview but without the rendering optimizations.

Reply:
Test if taps still work when using FlashList